site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md

```diff
@@ -0,0 +1,83 @@
+
+On the other hand, if the application is aimed at enterprise customers, integrating with ActiveDirectory can help adoption. If developers are your target market, GitHub authentication eases the sign-up process as well as signals to them that you understand their needs.
+
+If your auth system allows for multiple different identity providers, your application can meet users "where they are", rather than requiring them to create and manage a new account. 
+
+If you work for a large organization, you may need to federate your user management system, using one of the standards such as OIDC, to other user datastores. Having a central service for your suite of applications means federation and integration only needs to be done once, rather than for each application you build.
+
+## Challenges
+
+Of course, nothing is perfect. There are challenges with this approach as well. Some technical, some not so much. 
+
+First off is the [SSO tax](https://sso.tax/). Many third party applications don't support this user auth delegation until you are on an enterprise plan. Ouch. Investigate required applications to see if and how they can act as a RP or SP to a centralized user management system before you decide to pursue this architecture. 
+
+Another organizational challenge is ensuring developers and users don't do an end run around the organization's user management system. Encourage employees to work within these constraints by making it as easy as possible and repeatedly explaining the benefits. Providing examples of successful integrations can help with both goals.
+
+Tying together the bottleneck system, the delegating applications, and the external identity providers requires effort. Sometimes it is as simple as following a tutorial on a website and adding a few lines of configuration. Other times it may be more complicated and may require support.
+
+Finally, beware of insecure or slow auth services. No one cares about authentication and authorization, except when it doesn't work. When was the last time you heard someone exclaim "I love that login page!"? People want to log in when and how they choose and have it work, so they can get on to the task they need to accomplish. Select a system that is robust, has great support, and is flexible enough to meet future needs.
+
+## See It In Action
+
+If you'd like to see how easy it is to configure FusionAuth to serve as an centralized authentication and authorization service, HYPR and FusionAuth are hosting a webinar on Thursday, July 16 2020. 
+
+In less than 15 minutes, Zendesk will be set up as an application delegating auth decisions to FusionAuth (well, if you want to be technical, as an SP using SAML). HYPR will be configured as an identity provider. At the end, a user will be able to sign into Zendesk using HYPR's passwordless technology. [Sign up here!](https://get.hypr.com/fusionauth-webcast)
```

Comment:
Should we make the CTA a button rather than an inline link?

Reply:
Button made.